.github/workflows/ci.yml

```diff
@@ -19,16 +19,30 @@ jobs:
   build:
     runs-on: ubuntu-18.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
I'd rather pin it to `20.04` as the latest `20.10` isn't an LTS version.

Reply:
We're using "latest" in a couple of workflow files already and are gradually switching all over. It's just tedious to keep bumping this across all files, in addition to the ones we already have to bump (Node, etc)

Reply:
GitHub is suggesting to use latest in all their code examples, I think they are only offering LTS anyway:

> Operating System
> Ubuntu
> 20.04.5
> LTS

Reply:
Interesting that the CI fails with “latest" though...

Reply:
Bumping the Ubuntu version didn't fix Node 18 but the server runs `20.04` and it works fine. Puppeteer doesn't support Node 12 which we were going to remove anyway.

Reply:
So we'll stay on latest for now, it should work.

Reply:
Found [this](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) regarding "latest":

> Note: The -latest runner images are the latest stable images that GitHub provides, and might not be the most recent version of the operating system available from the operating system vendor.